rampwf/score_types/brier_score.py

```diff
@@ -0,0 +1,154 @@
+from .base import BaseScoreType
+import numpy as np
+from sklearn.metrics import brier_score_loss
+
+class BrierScore(BaseScoreType):
+    is_lower_the_better = True
+    minimum = 0.0
+    maximum = 1.0
+
+    def __init__(self, name='brier_score', precision=3):
```

Comment:
please add proper docstrings for the functions, methods and classes.

Here it should be

"""
Parameters
------------
name : str
     The human readable name for the score.
precision : int
     The floating point precision of ???
"""

we follow numpydoc formatting guidelines.

see Extended Summary section in : https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt